test/e2e/tests/confirmations/transactions/erc20-approve-redesign.spec.ts

```diff
@@ -85,11 +85,11 @@ async function importTST(driver: Driver) {
   await driver.clickElement('[data-testid="importTokens"]');
 
   await driver.waitForSelector({
-    css: '.import-tokens-modal__button-tab',
+    css: '[data-testid="import-tokens-modal-custom-token-tab"]',
     text: 'Custom token',
   });
   await driver.clickElement({
-    css: '.import-tokens-modal__button-tab',
+    css: '[data-testid="import-tokens-modal-custom-token-tab"]',
```

Comment:
Updating tests to use a new data test id selector instead of classname that was passed to the now removed `buttonClassName` tab prop